Updated article with more detailed info (#69870)
Azure/azure-content: articles/firewall-manager/threat-intelligence-settings.md

:::image type="content" source="media/threat-intelligence-settings/threat-intelligence-policy.png" alt-text="Threat intelligence policy":::

## Threat intelligence Mode
Threat intelligence can be configured in the following modes. By default, threat intelligence-based filtering is enabled in alert mode.

Mode |Description  |
|---------|---------|

> |`Off`     | The Threat Intelligence feature will not be enabled for your firewall |

Reviewer: ```suggestion
|`Off`     | The Threat Intelligence feature is not enabled for your firewall |
```

|`Alert only`     | You will receive high confidence alerts for traffic going through your firewall to or from known malicious IP addresses and domains |

> |`Alert and deny`     | Traffic will be blocked and you will receive high confidence alerts when traffic attempting to go through your firewall to or from known malicious IP addresses and domains is detected. |

Reviewer: ```suggestion
|`Alert and deny`     | Traffic is blocked and you will receive high confidence alerts when traffic is detected attempting to go through your firewall to or from known malicious IP addresses and domains. |
```
